sdks/java/core/pom.xml

```diff
@@ -732,6 +732,12 @@
       <version>1.0-rc2</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value</artifactId>
+      <version>1.1</version>
+      <scope>provided</scope>
```

Comment:
`provided` is better than `compile` here, as the dependency is not propagated transitively ([reference](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope)). `optional` is then no necessary since it deals with transitive dependencies.

Also, this is the [recommended pom.xml definition from AutoValue](https://github.com/google/auto/blob/master/value/userguide/index.md#in-pomxml).